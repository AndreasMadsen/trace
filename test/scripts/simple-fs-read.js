'use strict';

const fs = require('fs');

//
// test module method (fs.readFile)
//

fs.readFile(__filename, function callback() {
  throw new Error('trace');
});
