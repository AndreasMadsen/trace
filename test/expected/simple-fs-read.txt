/scripts/simple-fs-read.js:10
  throw new Error('trace');
  ^

Error: trace
    at callback (/scripts/simple-fs-read.js:10:9)
    at Object.<anonymous> (/scripts/simple-fs-read.js:9:4)
